modules/node_modules/@ncigdc/uikit/Banner.js

```diff
@@ -9,8 +9,9 @@ const style = {
     display: 'flex',
     alignItems: 'center',
     backgroundColor: '#265986',
-    color: 'white',
+    color: '#000000',
```

Comment:
nah the font color needed to be black (or some other really dark color)
the alternative would have been to darken the background color from yellow to a nasty looking brown